Update base OS info #268 (#298)
rockstor/rockstor-doc: installation/pre-install-howto.rst

Memory Test (memtest86+)
------------------------

The `memtest86+ <https://www.memtest.org/>`_ program is actually derived from
the linux kernel itself, the core of the operating system that Rockstor is
built on. Memtest86+ is licensed under the GPL.

Memtest86+ Cautionary Note
^^^^^^^^^^^^^^^^^^^^^^^^^^

Memtest86+ can place a very intense load on your system, especially if run in
the new Multithreaded mode (version 5.01 and onwards via F2 on initial start)
but a sufficiently cooled system should be able to execute this test
indefinitely; however, if your system has cooling issues then it may lockup

> or even sustain damage. Please take care to monitor your systems temperate

Reviewer: ```suggestion
or even sustain damage. Please take care to monitor your system's temperatures
```

> during this test. Version 5.01 and on have a built in CPU temperature monitor.

Reviewer: ```suggestion
during this test. Version 5.01 and on have a built-in CPU temperature monitor.
```

If you find that your version of the Rockstor installer doesn't include the 5
.01 or newer version of memtest86+, or any version at all (Rockstor 4) then
download a `Pre-Compiled Bootable ISO <https://www.memtest.org/#downiso>`_

> from the projects official page and write it to a USB key in exactly the same

Reviewer: ```suggestion
from the project's official page and write it to a USB key in exactly the same
```
